.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,27 @@
-Please delete this line and the text below before submitting your contribution.
+# Pull Request checklist
 
----
+Thanks for contributing!
+Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.
```

Comment:
I would rephrase this a bit, e.g. propose that people submit an issue first to discuss their idea before implementing it. That way you avoid disappointment and are sure that all contributions can actually be incorporated.

Reply:
@bouweandela thanks. I added a section "before you start" in the checklist.